appsrc/tasks/find-upload.js

```diff
@@ -48,6 +48,12 @@ let self = {
       score -= 100
     }
 
+    /* Native uploads are preferred */
+    if (upload.type === 'html') {
+      score -= 20
+      score -= 20
```

Comment:
that wasn't intended. the git history for this commit is pretty bad. i'll push a fix and remove [this commit](https://github.com/Mrmaxmeier/itch/commit/6aabc8072ec7e8c260923b6c85a21b25d37a170f).